exammode/templates/exammode/examsession_detail.html

```diff
@@ -0,0 +1,25 @@
+{% extends "exammode/base.html" %}
+{% load course %}
+{% load static %}
+
+{% block content %}
+<div class="text-center">
+	<p>
+		Please confirm that you selected the correct exam.
```

Comment:
Translate text.

Reply:
The "trans" tags are now added. Translations are still missing.